packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -789,3 +806,56 @@ export async function DocsQA<ChunkMetadata extends Jsonifiable = Jsonifiable>(pr
     </ChatCompletion>
   );
 }
+
```

Comment:
I like this approach! This is a neat way of getting the model to give a structured output so it can be fed into props of another component.